features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/block.jag

```diff
@@ -0,0 +1,8 @@
+<%
+jagg.block("loganalyzer", {
```

Comment:
Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD